CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.6]
+
+### Changed
+* Updated the metadata for the RGB Decomposition tool in the ArcGIS Toolbox to more accurately reflect behavior
```

Comment:
Should we also call out "cleans up some formatting and typos in all of the metadata xmls"?